wgengine/netstack/netstack.go

```diff
@@ -54,13 +54,20 @@ type Impl struct {
 	// port other than accepting it and closing it.
 	ForwardTCPIn func(c net.Conn, port uint16)
 
-	ipstack     *stack.Stack
-	linkEP      *channel.Endpoint
-	tundev      *tstun.Wrapper
-	e           wgengine.Engine
-	mc          *magicsock.Conn
-	logf        logger.Logf
-	onlySubnets bool // whether we only want to handle subnet relaying
+	// ProcessAll is whether netstack should handle all incoming traffic.
+	ProcessAll bool
+
+	// ProcessSubnets is whether netstack should traffic to
+	// non-local IPs (e.g. it should be a subnet router). This
+	// option is ignored ProcessAll is set anyway.
```

Comment:
"This option is ignored ProcessAll is set anyway."

Something got deleted from that sentence.

Reply:
Whoops. I'll rewrite. And might rename these fields. They're subtle and confusing.

Reply:
PTAL; renamed & rewritten.